pom.xml

```diff
@@ -82,6 +80,79 @@
                     <target>${java.version}</target>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.8.2</version>
+                <configuration>
+                    <locales>en</locales>
+                </configuration>
+            </plugin>
         </plugins>
+
     </build>
+
+     <scm>
+         <connection>scm:git:ssh://github.com/Decathlon/ara</connection>
+         <developerConnection>scm:git:ssh://github.com/Decathlon/ara</developerConnection>
+    </scm>
+
+    <distributionManagement>
+        <site>
+            <id>ara-site</id>
+            <!-- The site is not ready to be deployed yet. When ready, put the final url below (replacing: localhost/tmp) -->
+            <!-- For instance, if your site is www.my-site.com, then you should have the following: <url>scp://my-site.com</url> -->
+            <url>scp://localhost/tmp/</url>
+        </site>
+    </distributionManagement>
+
```

Comment:
It could be a great addition to add the following fields : 

* issueManagement
* project Informations (like a Homepage and a description which could redirect to this Github repo)
* a detailled member list of the team (and not just Decathlon, but like every member of the ARA-Team ?)

Reply:
updated !